test/one/profiler/test/TestProcess.java

```diff
@@ -66,6 +67,7 @@ public TestProcess(Test test, Os currentOs, String logDir) throws Exception {
         this.currentOs = currentOs;
         this.logDir = logDir;
         this.inputs = test.inputs();
+        this.test = test;
```

Comment:
Minor stylistic comment: things being equal, arguments, fields and initialization should follow the same order.